charts/linkerd2-multicluster/templates/gateway.yaml

```diff
@@ -96,7 +96,6 @@ metadata:
   namespace: {{.Values.namespace}}
   annotations: 
     mirror.linkerd.io/gateway-identity: {{.Values.gatewayName}}.{{.Values.namespace}}.serviceaccount.identity.{{.Values.linkerdNamespace}}.{{.Values.identityTrustDomain}}
-    mirror.linkerd.io/probe-port: "{{.Values.gatewayProbePort}}"
```

Comment:
is this no longer an annotation because it is now discovered from the ports in the gateway service spec?